Code/Numerics/testConrec.cpp

```diff
@@ -85,3 +85,177 @@ width='300px' height='300px' >
     delete[] grid;
   }
 }
+
+TEST_CASE("connectLineSegments", "[conrec]") {
+  SECTION("basics") {
+    std::vector<RDGeom::Point2D> pts = {{0., 0.}, {1., 0.}, {1., 1.}, {0., 1.}};
+    const size_t gridSz = 100;
+    auto *grid = new double[gridSz * gridSz];
```

Comment:
perhaps use std::vector<double> grid and use grid.data().

It's testing code I know ;)